### PR TITLE
feat: remove cosmwasm

### PR DESCRIPTION
**Pull request by audtlr24**
## Background
We decided it would be better to remove cosmwasm from the chain. There are three reasons for this
1. Hardship in removing CosmWasm once somebody starts using it; currently, nobody is using CosmWasm on Panacea yet
2. Possibility that anybody can deploy a contract that does not fit the purpose of Panacea
3. Replacable through Cosmos SDK v0.46.x that now includes x/nft module later

### CODE REVIEW

**Comment by audtlr24** [comment]
I haven't tested the upgrade handler yet. I will open the pr after testing is complete.

**Comment by audtlr24** [comment]
Upgrading directly from v2.0.5 had issues with the VersionMap part, so I tested the upgrade in branch `v2.0.5/add-set-version-map`.

**Comment by gyuguen** [comment]
I think you can remove to wasm config in Dockerfile

https://github.com/medibloc/panacea-core/blob/main/Dockerfile

* Line 8~9
* Line 20~21
* Line 28~29

**Comment by gyuguen** [comment]
And you can remove this part.
https://github.com/medibloc/panacea-core/blob/ft/na/remove-cosmwasm/.gitbook/overview/tech-stack.md?plain=1#L45
https://github.com/medibloc/panacea-core/blob/ft/na/remove-cosmwasm/README.md?plain=1#L15

**Review by gyuguen — APPROVED** [review]
LGTM

I tested this locally and didn't have any issues.
And I left a few comment :)

File: .gitbook/overview/tech-stack.md [hunk 1]
```diff
@@ -41,15 +41,6 @@ Also, data holders can reveal only data that they really want to share using the
...
 
-
-## Smart Contract
-
-Panacea provides the Smart Contract framework powered by CosmWasm.
-Any developers can implement their own smart contracts using the Rust programming language and deploy them on Panacea.
-It means that any fancy ideas such as DeFi and NFT can be implemented on Panacea,
-and they can be used for boosting the liquidity of health data flow of medical systems built on Panacea.
-
-
```
Comment (youngjoon-lee) [inline .gitbook/overview/tech-stack.md:52]: I updated a diagram as well: 15ad4233e4f30f15a697dbb24de1d731f76614d2

File: app/app.go [hunk 2]
```diff
@@ -848,6 +763,9 @@ func (app *App) registerUpgradeHandlers() error {
 	})
 
 	app.UpgradeKeeper.SetUpgradeHandler("v2.0.6", func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+
+		app.IBCKeeper.ConnectionKeeper.SetParams(ctx, ibcconnectiontypes.DefaultParams())
+
```
Comment (youngjoon-lee) [inline app/app.go:768]: Do we need to have this for the v2.0.6 upgrade handler?

Comment (audtlr24) [reply app/app.go]: Ah, we don't need it. I fix it, thanks!

File: .gitbook/overview/tech-stack.md (hunk 1 above)
Comment (audtlr24) [reply .gitbook/overview/tech-stack.md:52]: Thanks!

File: app/app.go (hunk 2 above)
Comment (youngjoon-lee) [reply app/app.go]: thanks!

**Review by youngjoon-lee — APPROVED** [review]
lgtm,man

**Review by inchori — APPROVED** [review]
LGTM!

**Review by H4NLee — APPROVED** [review]
LGTM

**Pull request merged**